server/src/domain/media/media.service.ts

```diff
@@ -49,6 +52,32 @@ export class MediaService {
       }
     }
 
+    const personList = force
+      ? await this.personRepository.getAll()
+      : await this.personRepository.getAllWithoutThumbnail();
+
+    for (const person of personList) {
+      // use stored asset for generating thumbnail or pick a random one if not present
+      const face = person.faceAssetId
+        ? await this.personRepository.getFaceById({ personId: person.id, assetId: person.faceAssetId })
+        : await this.personRepository.getRandomFace(person.id);
+      if (face) {
+        const jobData = {
+          imageWidth: face.imageWidth,
+          imageHeight: face.imageHeight,
+          boundingBox: {
+            x1: face.boundingBoxX1,
+            x2: face.boundingBoxX2,
+            y1: face.boundingBoxY1,
+            y2: face.boundingBoxY2,
+          } as BoundingBox,
```

Comment:
Why is this cast necessary? Also, can you just inline the job data into the `queue` call instead?